packages/lib/src/forms/dao/GenesisProtocolForm.ts

```diff
@@ -10,6 +10,7 @@ import {
   validAddress,
   validNumber,
   validPercentage,
+  validQueuedVotePercentage,
```

Comment:
It does not make sense to have a property name (Queued Vote Percentage) in the validation file. These validators are generic data validation utilities.